docs/lab/02/02.md

```diff
@@ -644,6 +644,70 @@ If the MCU provides timers, Embassy-rs will use them to suspend the software ins
 
 :::
 
+## Build and flash
+
+### Build
+
+To build a program for the Raspberry Pi Pico, run the following command in the crate's root folder (where the `Cargo.toml` file and `src` folder are):
+
+```shell
+cargo build
+
+
+This will create the binary file inside `target/thumbv6m-none-eabi/debug/`.
+
+:::note
+
+If the crate is part of a workspace (as it is the lab template), the `target` folder is located in the workspace's root instead of the crate's root.
+
+:::
+
+### Flash
+
+To load your program to the Raspberry Pi Pico, connect the board through USB to your computer while holding down the `BOOTSEL` button. It should appear as an external drive on your system. Afterwards, you can run the following command (also from the root folder):
+
+`elf2uf2-rs -d .\target\thumbv6m-none-eabi\debug\<project_name>`
+
+This will upload your program to the Pico.
+
+:::note
+
+If the above command doesn't work, try these steps instead:
+
+- run this command: `elf2uf2-rs .\target\thumbv6m-none-eabi\debug\<project_name>`
+
+- drag and drop the `.uf2` file located in the `.\target\thumbv6m-none-eabi\debug\` folder to the RP external drive
+
+:::
+
+### Serial console
+
+To see messages from the Pico over the serial port, add the `-s` argument to the flash command:
+
+`elf2uf2-rs -s -d .\target\thumbv6m-none-eabi\debug\<project_name>`
+
+This will allow us to see messages sent over serial from the board.
+
+### Troubleshooting
+
```

Comment:
```suggestion

#### Link error - reset vector is missing

#### Flash succeeds, the application does not start

```

Reply:
Added the two separate sections, and information on the reset vector error.